bigquery_etl/docs/mozdata/generate_mozdata_docs.py

```diff
@@ -0,0 +1,65 @@
+import os
+import yaml
+from pathlib import Path
+from jinja2 import Environment, FileSystemLoader
+
+VIEW_FILE = "view.sql"
+METADATA_FILE = "metadata.yaml"
+NON_USER_FACING_DATASET_SUFFIXES = (
+    "_derived",
+    "_external",
+    "_bi",
+    "_restricted",
+)
+SOURCE_URL = "https://github.com/mozilla/bigquery-etl/blob/master"
+
+
+def generate_mozdata_docs(out_dir, project_dir):
+    """Generate documentation for mozdata."""
+    project_doc_dir = Path(os.path.join(out_dir, "mozdata"))
+
+    data_sets = [
+        item
+        for item in os.listdir(project_dir)
+        if os.path.isdir(os.path.join(project_dir, item))
+        and all(name not in item for name in NON_USER_FACING_DATASET_SUFFIXES)
+    ]
+
+    for table in data_sets:
+        output = []
+        source_urls = {}
+
+        with open(project_doc_dir / f"{table}.md", "w") as dataset_doc:
+            # Manually set title to avoid Mkdocs from auto-formatting file names
+            dataset_doc.write(f"---\ntitle: {table}\n---\n\n")
+
+            for root, dirs, files in os.walk(f"{project_dir}{table}"):
+                if dirs == []:
+                    dataset_name = root.split("/")[-1]
+                    source_urls["Table"] = f"{SOURCE_URL}/{root}"
```

Comment:
```suggestion
                    source_urls["Table"] = f"{SOURCE_URL}/{root}"
```

This renders a link labeled "Table Source", which is confusing to me, since in most cases it's a view being defined rather than a table. This could perhaps be "Source Directory".

Reply:
I suppose "View Source" is also a bit problematic because it can be read as a verb, as in "click this link to view the source".

So perhaps the links can be `Source Directory | Metadata File | View Definition`